tests/python/gpu/test_operator_gpu.py

```diff
@@ -2300,6 +2300,81 @@ def test_math():
             for op in ops:
                 run_math(op, shape, dtype, check_value=check_value)
 
+@with_seed()
```

Comment:
One more blank line above, 2 blank lines between functions.

Reply:
Fixed, thank you.